### PR TITLE
Update UseLimitations

### PR DESCRIPTION
**Pull request by gregbunce**
and update links and wording in the readme.md

### CODE REVIEW

**Review by stdavis — APPROVED** [review]
Looks great to me. Thanks!

File: src/sweeper/sweepers/UseLimitations.html [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 </p>
 
 <p>
-    <a href="http://creativecommons.org/licenses/by/4.0/">
-        <img alt="Creative Commons License" src="https://i.creativecommons.org/l/by/4.0/88x31.png" style="border-width:0;" />
+    <a href="http://creativecommons.org/licenses/by/4.0/" rel="nofollow and ugc">
```
Comment (steveoh) [inline src/sweeper/sweepers/UseLimitations.html:6]: **issue**: `and` is not used to add multiple relationships. I'm starting to think that esri modified the content when uploading. We will have to check to make sure we don't foul up any checks we may do in the future but I think the following is more correct. 
```suggestion
    <a href="http://creativecommons.org/licenses/by/4.0/" rel="license nofollow noopener ugc">
```

File: src/sweeper/sweepers/UseLimitations.html [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+        <img alt="Creative Commons License" src="https://i.creativecommons.org/l/by/4.0/88x31.png" rel="nofollow and ugc" style="border-width:0;" />
```
Comment (steveoh) [inline src/sweeper/sweepers/UseLimitations.html:7]:
```suggestion
        <img alt="Creative Commons License" src="https://i.creativecommons.org/l/by/4.0/88x31.png" rel="license nofollow noopener ugc" style="border-width:0;" />
```

File: src/sweeper/sweepers/UseLimitations.html [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
     </a>
-    <br />This work is licensed under a <a href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.
+    <br />This work is licensed under a <a href="http://creativecommons.org/licenses/by/4.0/" rel="nofollow and ugc">Creative Commons Attribution 4.0 International License</a>.
```
Comment (steveoh) [inline src/sweeper/sweepers/UseLimitations.html:9]:
```suggestion
    <br />This work is licensed under a <a href="http://creativecommons.org/licenses/by/4.0/" rel="license nofollow noopener ugc">Creative Commons Attribution 4.0 International License</a>.
```

**Comment by gregbunce** [comment]
thanks for the correction.  let me know if there's anything else.  if not, it should be ready for a merge.

**Review by steveoh — APPROVED** [review]

**Comment by gregbunce** [comment]
@stdavis or @steveoh - it looks like this is ready to be merged.

**Pull request merged**